### ISSUE TITLE
RTL- Diposition of menu- News detail: Wrong disposition of menu

### ISSUE THREAD

**Issue by Jihed525** [Meeds-io/meeds#3559]
Environment: Tribe
Step of reproduction:
- Access to any News detail
- Click on the three points button on the top left of the page
Current behavior:
The menu appears on the left side of the button with bad display (see attachment)
Expected behavior:
On RTL mode, The menu option should appear on the right side of the button

builders:9549

**Comment by boubaker** [comment]
> see attachment

Missing attachment

**Comment by Jihed525** [comment]
> > see attachment
> 
> Missing attachment

@mbensalem attachement needed (wasn't found on original task)

**Comment by Jihed525** [comment]
attachment 

<img width="435" height="263" alt="Image" src="https://github.com/user-attachments/assets/e0e9d84f-86ec-4820-918d-6cdb90a847c5" />

**Comment by boubaker** [comment]
Thanks
